### PR TITLE
do not crash on maformed ymal

### PR DESCRIPTION
**Pull request by haim-kermany**
Signed-off-by: haim-kermany <haim.kermany@gmail.com>

### CODE REVIEW

File: nca/NetworkConfig/ResourcesHandler.py [hunk 1]
```diff
@@ -244,16 +245,22 @@ def _parse_resources_path(self, resource_list, resource_flags):
...
+                                self.pods_finder.add_eps_from_yaml(res_code)
+                                self.services_finder.namespaces_finder = self.ns_finder
+                                self.services_finder.parse_yaml_code_for_service(res_code, yaml_file)
+                            if ResourceType.Policies in resource_flags:
+                                self.policies_finder.parse_yaml_code_for_policy(res_code, yaml_file.path)
+
+                    except error.MarkedYAMLError as prs_err:
+                        print(
+                            f'{prs_err.problem_mark.name}:{prs_err.problem_mark.line}:{prs_err.problem_mark.column}:',
+                            'Parse Error:', prs_err.problem)
```
Comment (zivnevo) [inline nca/NetworkConfig/ResourcesHandler.py:263]:
```suggestion
                        print(
                            f'{prs_err.problem_mark.name}:{prs_err.problem_mark.line}:{prs_err.problem_mark.column}:',
                            'Parse Error:', prs_err.problem, file=stderr)
```

**Review by zivnevo — CHANGES_REQUESTED** [review]
Please also change the error you print in `nca_main()` to be printed to stderr.

File: nca/NetworkConfig/ResourcesHandler.py (hunk 1 above)
Comment (haim-kermany) [reply nca/NetworkConfig/ResourcesHandler.py]: done

Comment (zivnevo) [reply nca/NetworkConfig/ResourcesHandler.py]: Please also change the error you print in `nca_main()` to be printed to stderr.

Comment (haim-kermany) [reply nca/NetworkConfig/ResourcesHandler.py]: done

**Review by zivnevo — APPROVED** [review]
OK to Squash and Merge

**Pull request merged**